js2-mode.el

```diff
@@ -7092,14 +7074,33 @@ later. NODE must be a name node."
 We have to wait until entire buffer is parsed, since JavaScript permits var
 decls to occur after they're used.
 
-If any undeclared var name is in `js2-externs' or `js2-additional-externs',
-it is considered declared."
-  (let (name)
+Some identifiers may be assumed to be externally defined.
+These externs are not highlighted, even if there is no declaration
+for them in the source code.
+The list of externs consists of the following:
+- the list `js2-ecma262-externs' for basic names from the ECMAScript language standard
+- depending on the buffer-local variables `js2-include-?-externs'
+  the corresponding `js2-?-externs' to add names for certain environments
+  like the browser or node/rhino
+- two user-customizable lists `js2-global-externs' and `js2-additional-externs'
+  of which the latter is a buffer-local variable
+
+See especially `js2-additional-externs' for further details about externs."
+  (let ((default-externs
+          (append js2-ecma-262-externs
```

Comment:
`js2-ecma-262-externs` or `js2-browser-externs` should probably go last here (the last argument is not copied).

This function is probably fast enough anyway, but a tiny speed boost wouldn't hurt.